src/assets/SearchNotFoundIcon.tsx

```diff
@@ -0,0 +1,18 @@
+const SearchNotFoundIcon = () => (
+  <svg
+    width='181'
+    height='250'
```

Comment:
```suggestion
    width='11.3125rem'
    height='15.625rem'
```

반응형을 위해서 이 부분 수정 부탁합니다

Reply:
넵 반영하겠습니다!